informer/src/handler/subscription.rs

```diff
@@ -14,28 +14,41 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{ColdEvents, EventTags, Events, Params, Sink};
+use crate::{ColdEvents, EventTags, Events, Params, Sink, SubscriptionId};
 use jsonrpc_core::futures::Future;
+use std::sync::atomic::AtomicBool;
+use std::sync::Arc;
 
 #[derive(Clone)]
 enum ConnectionState {
     Connected,
 }
 
+#[derive(Clone)]
+pub enum SubscriptionState {
+    Subscribed,
+    UnSubscribed,
+}
```

Comment:
I couldn't understand the role of the `SubscriptionState`.
Could you explain it to me?

Reply:
@majecty it checks whether the subscription has a cold event or not.

Reply:
Do we need to check it? When do we need to do distinguish them?

Reply:
@majecty I realized it is not necessary to have `SubscriptionState`.